interface/src/index.html

```diff
@@ -70,6 +70,10 @@
                 Download
               </a>
             </li>
+
+            <login-button></login-button>
+            <profile-button></profile-button>
```

Comment:
I agree with that - it does seem much more intuitive to have the main AppCtrl make a request for the User, and then have the ng-if logic here where these directives are inserted.